public/index.html

```diff
@@ -5,7 +5,7 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width,initial-scale=1.0">
     <link rel="icon" href="<%= BASE_URL %>favicon.ico">
-    <title>israeli-bank-scrapers-desktop</title>
+    <title>Budget tracking</title>
```

Comment:
Do you want to think about less generic name?
The name from my repo was "Oshi"-"עו"שי".
Can't say I'm in love with this name, but I just think if we want another name, we need to decide now, before public release.